src/main/scala/com/fulcrumgenomics/cmdline/FgBioMain.scala

```diff
@@ -73,6 +81,11 @@ class FgBioCommonArgs
   System.setProperty("java.io.tmpdir", tmpDir.toAbsolutePath.toString)
 
   Logger.level = this.logLevel
+
+  if (FgBioCommonArgs.intelSupported) {
```

Comment:
I think I'd argue that this code probably belongs in `FgBioMain` since it's not controlled/controllable via a parameter in `FgBioCommonArgs`.